### ISSUE TITLE
Homework 04 ready for grading

### ISSUE THREAD

**Issue by burkeprw** [burkeprw/STAT545-hw-burke-patrick#5]

**Comment by HScheiber** [comment]
### Main Repository/README:

- You've included all of your homeworks into a single repo without sub-README files, but you make it work.

- Very thorough and well formatted progress report.

### Tidy data and joins

#### Assignment 1: General data reshaping and relationship to aggregation

- You have two titles `hw04_tidy_data` and `Homework 04: Tidy data and joins`. I would recommend just using the one.

- It's cool that you used your own data set. Must have made things more challenging!

- It's hard to follow exactly what your data is. It would have been nice if you explained a bit more here. I had to go to your previous assignment to figure it out.

- I think when you publish your code, you should remove the commented out bits of code (of course leave the comments intact). Leaving it in only serves to confuse the reader.

- You should include outputs of the data structures at intermediate steps, I think it would help the reader follow along better.

- Definitely utilize the `head` function to truncate excessively long tables.

- For plots, think about using a more descriptive x axis label.  Also try modifying `theme` elements in `ggplot` for nicer looking plots.

#### Assignment 2: Join, merge, look up

- Reading through this left me scratching my head. You've got to include more explanations in your work. 

- Again, you should truncate your outputs. There's no reason to include an output with 100's of rows.

- It would be nice if you defined your acronyms somewhere. As it is, I found it very hard to understand what you're doing with your data.

- Work on labelling of plots. Your x-axis tick labels are unreadable and your legend/axis labels don't provide any information. 

Overall a good job. Obviously a lot of work put into it, but when you do these assignments think about your audience please!

**Comment by derekcho** [comment]
Hi @burkeprw, here are some comments about your hw04:
-	It isn’t really apparent what your original dataset looks like, nor what your first set of transformations does to it. I think that it would be a good idea to show a subset of the data at each step. Likewise, you never show the effects of the gather function on your wide table, this is also important since readers shouldn’t have to run the code themselves to see what the results are!
-	Likewise in task 2, showing the entire dataset is not necessarily useful for explaining the different types of joins (a small subset will do). Conversely, you should show something rather than nothing for your second dataset sites_fwcp so readers can get an idea of what your data looks like and how it relates to your first dataset without having to dig through your code.
-	You should explore other types of joins rather than just the left join and show the results of the joins in a table
-	Using your own dataset is fine, and we want to encourage students to do things that are novel and that they find interesting! It definitely adds another level of complexity to the task at hand and it is great that you challenged yourself to do something different. However, you should still focus on the main points of the assignment which are to demonstrate the usage of spread, gather, and different types of joins. What does your gathered dataset look like? What does your joined dataset look like? What does the data look like before you applied these functions?

Your grade will be emailed to you at a later date.
